tardis/tests/tests_slow/test_w7.py

```diff
@@ -11,6 +11,8 @@
 from tardis.io.config_reader import Configuration
 
 
+@pytest.mark.skipif(not pytest.config.getvalue("integration-tests"),
+                    reason="integration tests are not included in this run")
```

Comment:
Should I keep this marker here or in `integration_tests_config` fixture ? I cannot keep it while unpacking it in `pytest_configure`, hence removed it from there.